packages/components-react/color-sample-react/src/color-sample.tsx

```diff
@@ -3,14 +3,15 @@ import { clsx } from 'clsx';
 import { forwardRef, useId } from 'react';
 
 interface ColorSampleProps extends HTMLAttributes<SVGSVGElement> {
+  label?: string;
   value: CSSProperties['color'];
 }
 
 export const ColorSample = forwardRef<SVGSVGElement, ColorSampleProps>(function ColorSampe(props, forwardedRef) {
   const id = useId();
-  const { className, style, title, value, ...restProps } = props;
-  const hasTitle = typeof title === 'string';
-  const ariaLabelledBy = [hasTitle ? id : undefined, props['aria-labelledby']].filter(Boolean).join(' ') || undefined;
+  const { className, label, style, value, ...restProps } = props;
+  const hasLabel = typeof label === 'string';
+  const ariaLabelledBy = [hasLabel ? id : undefined, props['aria-labelledby']].filter(Boolean).join(' ') || undefined;
```

Comment:
Als `aria-labelledby` is ingesteld, dan hoeft de `title` niet meer geassocieerd te zijn.